README.md

```diff
@@ -1,2 +1,40 @@
-# nlp-in-10-lines
-Repository for PyCons 2016 workshop Natural Language Processing in 10 Lines of Code.
+### PyCon UK 2016 
+![NLP in 10 Lines of Code](http://i.imgur.com/TuusTAm.png)
+# Natural Language Processing in 10 Lines of Code
+##### from Andraz Hribernik, Matic Horvat, Aeneas Wiener, Tom Sherborne
```

Comment:
I am not sure we really need to have authors specified (usually contributors on Github are shown with commits), but if @aeneaswiener and @matichorvat like it, I do not mind.